### DISCUSSION TITLE
Üçüncü ödev ve sonraki ödevler

### DISCUSSION THREAD

**Discussion by guidesoftware** [gusanmaz/BMB212-Algorithms-2223#24] [Q&A] [answered]
Üçüncü ödevden itibaren 3.02 ortalamayla uğraşmamızı istemişsiniz. bu 3.02 notu sabit bir not mu yoksa terminalden 2.12 gibi rastgele kodlar da deneyecek misiniz? ona göre bizde terminalden veri çekelim. ben sabit sayı olarak algılayıp 3.02'yı kullandım. yani 3.02 sayısı 3.4. ve 5. ödevde sabit statik bir şekilde kullanılması gerekiyor. terminalden değişken olarak ortalama çekmeyeceğiz.

**Comment by gusanmaz** [comment, marked as answer]
Odev aciklamalarinda tam olarak ne isteniyorsa onu yapmaniz gerekiyor.

Ornegin 4. odevde gecen bir cumle:

> Bu odevde ise komut satiri argumani ile okunan CSV dosyasindaki not ortalamasi 3.02 olan ogrencilerin sayisini yazdiracaktir.

Komut satiri argumani olarak GPA degeri okumanizi da isteyebilirdim, dosya yolu okumanizi istedigim gibi. Ama su ana kadar verdigim odevlerde boyle bir sey istemedim.
